ethereum/spec/src/test/java/tech/pegasys/teku/spec/logic/versions/deneb/block/BlockProcessorDenebTest.java

```diff
@@ -79,7 +80,7 @@ public void shouldCreateNewPayloadRequest() throws BlockProcessingException {
             .collect(Collectors.toList());
 
     final NewPayloadRequest newPayloadRequest =
-        spec.getBlockProcessor(UInt64.ONE).computeNewPayloadRequest(blockBody);
+        spec.getBlockProcessor(UInt64.ONE).computeNewPayloadRequest(preState, blockBody);
 
     assertThat(newPayloadRequest.getExecutionPayload())
```

Comment:
We could add an assertion here on `newPayloadRequest.getParentBeaconBlockRoot()`

Reply:
right good catch